frontend/packages/dev-console/src/components/taskruns/TaskRunDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { SectionHeading, ResourceSummary } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export interface TaskRunDetailsProps {
+  obj: K8sResourceKind;
+}
+
+const TaskRunDetails: React.FC<TaskRunDetailsProps> = ({ obj: taskRun }) => {
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text="Task Run Overview" />
```

Comment:
```suggestion
      <SectionHeading text={`${TaskRunModel.label} Overview`} />
```
We should endeavour not to manually write the model name out as text on the page.

Reply:
Done.